### PR TITLE
event calendar update to the latest version for elgg 1.12

### PR DESCRIPTION
**Pull request by Phanoix**
event calendar update to 1.10 version

### CODE REVIEW

File: mod/event_calendar/CHANGES.txt [hunk 1]
```diff
@@ -9,15 +9,66 @@ GC Changes
...
+- Updated code for sending of notifications to new notification system introduced with Elgg 1.9
+- UI adjustments for Aalborg theme (still works with default Elgg theme)
+- Code updates to reflect API changes of Elgg 1.9 including fixing of deprecation issues newly introduced by Elgg 1.9
+- Don't show sidebar menu entries for managing event subscriptions and requests if user has no permission to edit the corresponding event
+- Updated to be compatible with Widget Manager plugin for Elgg 1.9
 
-=== 1.8.0 (iionly and Juho Jaakola) ===
+1.8.0 (iionly and Juho Jaakola)
+-------------------------------
+>>>>>>> 1.10.3
```
Comment (nschonni) [inline mod/event_calendar/CHANGES.txt]: Looks like a merge issue

Comment (Phanoix) [inline mod/event_calendar/CHANGES.txt]: thanks, removed.

**Comment by ToferC** [comment]
Is this closed?

**Comment by Phanoix** [comment]
This didn't seem to change much, or at least didn't solve any of the issues with we had with it, closing this and upgrading straight to the elgg 2.x version in the upgrade branch.

**Pull request closed without merging**